File: Assignments/Assignment_12/p4.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=1;i<=row;i++){

                for(int j=0;j<row;j++){

                        printf("%d\t",(i*i));
                }
                printf("\n\n");
        }
}

File: Flutter/Assignment/Square pattern set-2/p4.dart
import "dart:io";
void main(){
	print("Enter the no. of rows:");
	int? row=int.parse(stdin.readLineSync()!);
	int x=1;
	for(int i=1;i<=row;i++){
		for(int j=1;j<=row;j++){
			if((j+i)%2==0){
				stdout.write("*\t");
			}else{
				stdout.write("${x}\t");
				x++;
			}
		}
		print("");
	}
}
